staging/src/k8s.io/client-go/util/flowcontrol/backoff.go

```diff
@@ -75,8 +95,9 @@ func (p *Backoff) Next(id string, eventTime time.Time) {
 	if !ok || hasExpired(eventTime, entry.lastUpdate, p.maxDuration) {
 		entry = p.initEntryUnsafe(id)
 	} else {
-		delay := entry.backoff * 2 // exponential
-		entry.backoff = time.Duration(integer.Int64Min(int64(delay), int64(p.maxDuration)))
+		// Exponential backoff, with a cap. Includes jitter from previous iterations.
+		baseDelay := time.Duration(integer.Int64Min(int64(entry.backoff*2), int64(p.maxDuration)))
```

Comment:
The old code had this problem too, but personally I find all the casts to be far worse than the `if` statement that `integer.Int64Min` lets us avoid.